cudet/updates.py

```diff
@@ -78,10 +78,10 @@ def take_action(self, parsed_args):
                 except ServerDataException as e:
                     if 'There is no deployment info for this environment' in e.message:
                         settings_action.download(parsed_args)
-                        _update_settings_file('/root/settings_{0}'.format(env_id))
+                        self._update_settings_file('/root/settings_{0}'.format(env_id))
                 else:
                     for filename in glob.glob("/root/deployment_{0}/*.yaml".format(env_id)):
-                        _update_settings_file(filename, full=False)
+                        self._update_settings_file(filename, full=False)
                 settings_action.upload(parsed_args)
                 env_redeploy = environment.EnvRedeploy()
```

Comment:
Please change `
env_redeploy = environment.EnvRedeploy()
env_redeploy.take_action(parsed_args)
`
to
`
                task_id = self.client.redeploy_changes(parsed_args.env_id)
                msg = ('Successfully prepared and started updates to '
                       'cluster {0} as task {1}'.format(parsed_args.env_id,
                                                        task_id))`